plugins/quay/src/components/QuayTagDetails/component.tsx

```diff
@@ -42,10 +42,24 @@ const columns: TableColumn[] = [
         </div>
       );
     },
+    customSort: (a, b) => {
```

Comment:
Maybe `(a: Vulnerability, b: Vulnarability) => {`?  It'd make for less code in the function too.

Reply:
Typescript would complain unforunately so I had to do some hacky type assertions
```
Type '(a: Vulnerability, b: {}) => number' is not assignable to type '(data1: {}, data2: {}, type: "row" | "group") => number'.
  Types of parameters 'a' and 'data1' are incompatible.
    Type '{}' is missing the following properties from type 'Vulnerability': Severity, NamespaceName, Link, FixedBy, and 3 more.ts(2322)
index.d.ts(185, 3): The expected type comes from property 'customSort' which is declared here on type 'TableColumn<{}>'
```

Reply:
Gotcha.  I see this could be something that could be addressed in the upstream interface if it doesn't accept type arguments, ideally this containing type could be a `TableColumn<Vulnerability>`  which could then supply the correct type for customSort to accept.  We'll have accept the approach you've taken as a workaround for now.